catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/CollectionDAO.java

```diff
@@ -357,6 +357,18 @@ int insert(
         @Bind("relation") int relation,
         @Bind("toEntity") String toEntity);
 
+    @SqlQuery(
+        "SELECT toId FROM entity_relationship "
+            + "WHERE fromId = :fromId AND fromEntity = :fromEntity AND relation = :relation AND toEntity = :toEntity "
+            + "AND (deleted = :deleted OR :deleted IS NULL) "
+            + "ORDER BY toId")
+    List<String> findTo(
+        @Bind("fromId") String fromId,
+        @Bind("fromEntity") String fromEntity,
+        @Bind("relation") int relation,
+        @Bind("toEntity") String toEntity,
+        @Bind("deleted") Boolean deleted);
+
```

Comment:
IT would be good get rid of other variant of methods that don't include `deleted` parameter and fix all the compiler warnings to move everything one shot to include `deleted` parameter to save time.

Reply:
I'm on it.

Reply:
Done.